composer/workflows/dataflowtemplateoperator_create_dataset_and_table_test.py

```diff
@@ -0,0 +1,63 @@
+from google.cloud import bigquery
+import pytest
+import google.cloud.exceptions
+import uuid
+
+client = bigquery.Client()
+dataset_UUID = str(uuid.uuid4()).split("-")[0]
+dataset_id = f"{client.project}.sample_dataset_{dataset_UUID}"
```

Comment:
I'm little bit confused about what this test is testing. I mean, it seems to me that this test is testing the client library, not the sample code.

I think this test should import some functions from `dataflowtemplateoperator_create_dataset_and_table_helper.py` and test its behavior.

Reply:
Yeah, I understand your concern here. @leahecole, when we discussed this test I know we discussed creating fixtures that would create and delete the datasets and tables. But, to @tmatsuo's point, this file essentially parallels the file it is attempting to test. What path would you recommend we take?

Reply:
I talked to @tmatsuo in a different chat and it seems like this is the case where we should make it so that we can do what he was suggesting - I'll follow up in 1-1 message to talk about strategy :)